mne/preprocessing/ica.py

```diff
@@ -244,11 +244,15 @@ class ICA(ContainsMixin):
               ii) additional noise reduction, it is a matter of current debate
               whether pre-ICA dimensionality reduction could decrease the
               reliability and stability of the ICA, at least for EEG data and
-              especially during preprocessing [5]_.
+              especially during preprocessing [5]_. (But see also [6]_ for a
+              possibly confounding effect of the different whitening/sphering
+              methods used in this paper (ZCA vs. PCA).)
+              sphering is more commonly called ZCA
```

Comment:
Start of a new sentence, right?

Reply:
Ah, I forgot to clean that last line, it's an artefact from the composing process ;) Thanks. Or do you think it's worth to mention the nomenclatura confusions surrounding whitening/sphering/PCA/ZCA? I wouldn't do that here, is probably too long already anyway, right?

Reply:
yes. Just remove these lonely words and +1 for MRG